apps/src/blockly/addons/cdoUtils.ts

```diff
@@ -335,6 +359,7 @@ export function locationField(icon, onClick) {
         // Just ignore bad values
       }
     }
+    return '';
```

Comment:
Could this have also been placed in the catch?

Reply:
No because we surround the `try/catch` with an `if`, so I put this at the end to cover both the catch and the else case.

Reply:
Oh, got it!